README.md

```diff
@@ -35,6 +35,41 @@ Example stream url:
 
 `clsp://172.28.12.57:9001/FairfaxVideo0520`
 
+### Tokenization
+
+With the latest version of the CLSP plugin, you are able to control stream
+access with two diferent token methods.
+
+#### JWT
+
+JSON Web Tokens provide the ability to attach a payload with authentication.
+To use JWT, use the following URI format:
+
+`[clsp protocol]-jwt://[sfs-ip-address]:[port-number-of-web-socket]/[stream-id]?token=[jwt-token]`
```

Comment:
When authenticating with hash, you prefix your url with `clsp-hash`. In the future when we support JWT, you would prefix with `clsp-jwt`. Work was done to add support for JWT but it was never completed. Should we remove all documentation references to JWT since it is not something we currently support?